src/storage/SDL_storage.c

```diff
@@ -160,7 +162,9 @@ SDL_Storage *SDL_OpenUserStorage(const char *org, const char *app, SDL_Propertie
             }
         }
     }
-    if (!storage) {
+    if (storage) {
+        SDL_LogBackend("user_storage", "generic");
```

Comment:
We have both Steam Cloud and console backends here, so we should check the bootstrap string maybe?

Reply:
Whoops, I think I pasted this from the asyncio code (which hardcodes "generic" on one of the codepaths) and forgot to update it. Good catch, I'll fix this!